app/components/SuccessMessage.tsx

```diff
@@ -0,0 +1,47 @@
+// LIBRARY IMPORTS
+import { Box, Button, Icon, IconButton, Text, VStack } from "@chakra-ui/react";
+import { FaRegPaperPlane, FaXmark } from "react-icons/fa6";
+
+// TYPE DEFINITIONS
+type SuccessMessageProps = {
+  handleCloseSuccess: () => void;
+};
+
+// SUCCESS MESSAGE COMPONENT DEFINITION
+export default function SuccessMessage({
+  handleCloseSuccess,
+}: SuccessMessageProps) {
+  return (
+    <Box position="relative" py={16}>
+      <IconButton
+        aria-label="close success message"
```

Comment:
Great idea to provide a way for the user to return to the request form. Though I noticed that "close" button and "Submit another request" button do the same thing. Do you think we need both? I vaguely remember that it might have something to do with accessibility. 
Also, should we consult Kayla on this and check if she has any feedback since it was not in the original design?

Reply:
I did intentionally make the buttons do the same thing, but mostly based on my intuition around UI experience, and not based any concrete reason :) Since all users might not want to submit another quote request, I also included the 'x' button to alert users to the fact that this is a closable message that has appeared in response to an action as a confirmation